gradle/dependencies.gradle

```diff
@@ -7,12 +7,12 @@ ext {
   ]
 
   version = [
-      mapboxMapSdk              : '9.6.0-alpha.1',
+      mapboxMapSdk              : '9.6.0-beta.1',
       mapboxSdkServices         : '5.8.0-beta.2',
       mapboxEvents              : '6.2.1',
       mapboxCore                : '3.1.0',
-      mapboxNavigator           : '25.0.0',
-      mapboxCommonNative        : '7.1.1',
+      mapboxNavigator           : '26.0.0',
+      mapboxCommonNative        : '9.0.2',
```

Comment:
Why do we named it mapboxCommon**Native**? a bit confused. it should be just mapboxCommon or mapboxCommonLibrary. wdyt?

Reply:
There are a bunch of `common`s 😅

Reply:
I see IO only https://github.com/mapbox/mapbox-navigation-android/pull/3677/files#diff-02b139e755ab0e27e0b0a6f9845843ef189116955cc340c49a4022587dbb2b52R38 🤔

Reply:
```
+--- org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.0
|    \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.0 -> 1.4.10
|         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.10
|         \--- org.jetbrains:annotations:13.0 -> 16.0.1
+--- project :libnavigation-ui
|    +--- project :libnavigation-core
|    |    +--- com.mapbox.base:common:0.4.0
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.10
|    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.10 (*)
|    |    |    +--- com.mapbox.base:annotations:0.4.0
|    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.10 (*)
|    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    +--- project :libnavigation-base
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-android-core:3.1.0
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-directions-models:5.8.0-beta.2
|    |    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.8.0-beta.2
|    |    |    |    |    \--- com.google.code.gson:gson:2.8.6
|    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-core:5.8.0-beta.2
|    |    |    |    +--- com.google.code.gson:gson:2.8.6
|    |    |    |    +--- com.squareup.retrofit2:retrofit:2.7.2
|    |    |    |    |    \--- com.squareup.okhttp3:okhttp:3.14.7 -> 3.12.10
|    |    |    |    |         \--- com.squareup.okio:okio:1.15.0 -> 2.4.3
|    |    |    |    |              +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.61 -> 1.4.10 (*)
|    |    |    |    |              \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.3.61 -> 1.4.10
|    |    |    |    +--- com.squareup.retrofit2:converter-gson:2.7.2
|    |    |    |    |    +--- com.squareup.retrofit2:retrofit:2.7.2 (*)
|    |    |    |    |    \--- com.google.code.gson:gson:2.8.5 -> 2.8.6
|    |    |    |    \--- com.squareup.okhttp3:logging-interceptor:3.12.7 -> 3.12.10
|    |    |    |         \--- com.squareup.okhttp3:okhttp:3.12.10 (*)
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10
|    |    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.10 (*)
|    |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.10 (*)
|    |    +--- com.mapbox.common:common:7.1.1
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.4.1 -> 5.8.0-beta.2 (*)
|    |    |    +--- com.mapbox.base:common:0.4.0 (*)
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    +--- com.mapbox.base:annotations:0.4.0 (*)
|    |    |    +--- com.mapbox.common:logger:0.4.0
|    |    |    |    +--- com.mapbox.base:common:0.4.0 (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.10 (*)
|    |    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.10 (*)
|    |    +--- com.mapbox.base:annotations:0.4.0 (*)
|    |    +--- project :libnavigation-util
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-directions-models:5.8.0-beta.2 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.0 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
|    |    |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.0 -> 1.4.10 (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9
|    |    |    |         \--- org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.3.9
|    |    |    |              +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.0 -> 1.4.10 (*)
|    |    |    |              \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.0 -> 1.4.10
|    |    |    \--- androidx.core:core:1.2.0
|    |    |         +--- androidx.annotation:annotation:1.1.0
|    |    |         +--- androidx.lifecycle:lifecycle-runtime:2.0.0 -> 2.2.0
|    |    |         |    +--- androidx.lifecycle:lifecycle-common:2.2.0
|    |    |         |    |    \--- androidx.annotation:annotation:1.1.0
|    |    |         |    +--- androidx.arch.core:core-common:2.1.0
|    |    |         |    |    \--- androidx.annotation:annotation:1.1.0
|    |    |         |    \--- androidx.annotation:annotation:1.1.0
|    |    |         +--- androidx.versionedparcelable:versionedparcelable:1.1.0
|    |    |         |    +--- androidx.annotation:annotation:1.1.0
|    |    |         |    \--- androidx.collection:collection:1.0.0 -> 1.1.0
|    |    |         |         \--- androidx.annotation:annotation:1.1.0
|    |    |         \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- project :libnavigator
|    |    |    +--- com.mapbox.navigator:mapbox-navigation-native:25.0.0
|    |    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.2.1 -> 5.8.0-beta.2 (*)
|    |    |    |    +--- com.mapbox.common:common:7.1.0 -> 7.1.1 (*)
|    |    |    |    +--- com.mapbox.common:okhttp:7.1.0 -> 7.1.1
|    |    |    |    |    +--- com.mapbox.common:common:7.1.1 (*)
|    |    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    |    +--- com.squareup.okhttp3:okhttp:3.12.12 -> 3.12.10 (*)
|    |    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.10 (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72 -> 1.4.10 (*)
|    |    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.8.0-beta.2 (*)
|    |    |    +--- project :libnavigation-base (*)
|    |    |    +--- project :libnavigation-util (*)
|    |    |    +--- com.mapbox.base:common:0.4.0 (*)
|    |    |    +--- com.squareup.okhttp3:okhttp:3.12.10 (*)
|    |    |    +--- com.squareup.okhttp3:logging-interceptor:3.12.10 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9 (*)
|    |    +--- com.mapbox.common:okhttp:7.1.1 (*)
|    |    +--- project :libnavigation-metrics
|    |    |    +--- project :libnavigation-base (*)
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-android-telemetry:6.2.1
|    |    |    |    +--- com.squareup.okhttp3:okhttp:3.12.0 -> 3.12.10 (*)
|    |    |    |    +--- com.google.code.gson:gson:2.8.5 -> 2.8.6
|    |    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |    +--- com.mapbox.mapboxsdk:mapbox-android-core:3.1.0
|    |    |    |    \--- androidx.legacy:legacy-support-core-utils:1.0.0
|    |    |    |         +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |         +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    |         +--- androidx.documentfile:documentfile:1.0.0
|    |    |    |         |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |         +--- androidx.loader:loader:1.0.0
|    |    |    |         |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |         |    +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    |         |    +--- androidx.lifecycle:lifecycle-livedata:2.0.0 -> 2.2.0
|    |    |    |         |    |    +--- androidx.arch.core:core-runtime:2.1.0
|    |    |    |         |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |         |    |    |    \--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    |         |    |    +--- androidx.lifecycle:lifecycle-livedata-core:2.2.0
|    |    |    |         |    |    |    +--- androidx.lifecycle:lifecycle-common:2.2.0 (*)
|    |    |    |         |    |    |    +--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    |         |    |    |    \--- androidx.arch.core:core-runtime:2.1.0 (*)
|    |    |    |         |    |    \--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    |         |    \--- androidx.lifecycle:lifecycle-viewmodel:2.0.0 -> 2.2.0
|    |    |    |         |         \--- androidx.annotation:annotation:1.1.0
|    |    |    |         +--- androidx.localbroadcastmanager:localbroadcastmanager:1.0.0
|    |    |    |         |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |         \--- androidx.print:print:1.0.0
|    |    |    |              \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    +--- project :libnavigation-util (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9 (*)
|    |    +--- com.mapbox.mapboxsdk:mapbox-navigation-accounts-android:0.4.0
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-android-accounts:0.8.0
|    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    \--- androidx.annotation:annotation:1.1.0
|    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-turf:5.8.0-beta.2
|    |    |    \--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.8.0-beta.2 (*)
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
|    |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9 (*)
|    |    +--- project :libnavigation-router
|    |    |    +--- project :libnavigation-base (*)
|    |    |    +--- com.mapbox.base:common:0.4.0 (*)
|    |    |    +--- project :libnavigator (*)
|    |    |    +--- project :libnavigation-util (*)
|    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-services:5.8.0-beta.2
|    |    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-core:5.8.0-beta.2 (*)
|    |    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.8.0-beta.2 (*)
|    |    |    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-directions-models:5.8.0-beta.2 (*)
|    |    |    |    \--- com.mapbox.mapboxsdk:mapbox-sdk-directions-refresh-models:5.8.0-beta.2
|    |    |    |         +--- com.mapbox.mapboxsdk:mapbox-sdk-directions-models:5.8.0-beta.2 (*)
|    |    |    |         \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    +--- com.squareup.okhttp3:okhttp:3.12.10 (*)
|    |    |    +--- com.squareup.okhttp3:logging-interceptor:3.12.10 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
|    |    |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9 (*)
|    |    |    \--- com.squareup.okio:okio:2.4.3 (*)
|    |    +--- project :libtrip-notification
|    |    |    +--- project :libnavigation-base (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
|    |    |    +--- project :libnavigation-util (*)
|    |    |    +--- androidx.core:core:1.2.0 (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9 (*)
|    |    \--- com.mapbox.common:logger:0.4.0 (*)
|    +--- com.mapbox.mapboxsdk:mapbox-android-sdk:9.6.0-alpha.1
|    |    +--- com.mapbox.mapboxsdk:mapbox-android-telemetry:6.2.0 -> 6.2.1 (*)
|    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-geojson:5.3.0 -> 5.8.0-beta.2 (*)
|    |    +--- com.mapbox.mapboxsdk:mapbox-android-gestures:0.7.0
|    |    |    +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    +--- com.mapbox.mapboxsdk:mapbox-android-accounts:0.7.0 -> 0.8.0 (*)
|    |    +--- com.mapbox.mapboxsdk:mapbox-android-sdk-gl-core:5.1.0
|    |    +--- com.mapbox.mapboxsdk:mapbox-sdk-turf:5.3.0 -> 5.8.0-beta.2 (*)
|    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    +--- androidx.fragment:fragment:1.0.0 -> 1.2.0
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    |    +--- androidx.collection:collection:1.1.0 (*)
|    |    |    +--- androidx.viewpager:viewpager:1.0.0
|    |    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |    +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    |    \--- androidx.customview:customview:1.0.0
|    |    |    |         +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |         \--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    +--- androidx.loader:loader:1.0.0 (*)
|    |    |    +--- androidx.activity:activity:1.1.0
|    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    |    |    +--- androidx.lifecycle:lifecycle-runtime:2.2.0 (*)
|    |    |    |    +--- androidx.lifecycle:lifecycle-viewmodel:2.2.0 (*)
|    |    |    |    +--- androidx.savedstate:savedstate:1.0.0
|    |    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    |    +--- androidx.arch.core:core-common:2.0.1 -> 2.1.0 (*)
|    |    |    |    |    \--- androidx.lifecycle:lifecycle-common:2.0.0 -> 2.2.0 (*)
|    |    |    |    \--- androidx.lifecycle:lifecycle-viewmodel-savedstate:1.0.0
|    |    |    |         +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |         +--- androidx.savedstate:savedstate:1.0.0 (*)
|    |    |    |         +--- androidx.lifecycle:lifecycle-livedata-core:2.2.0 (*)
|    |    |    |         \--- androidx.lifecycle:lifecycle-viewmodel:2.2.0 (*)
|    |    |    +--- androidx.lifecycle:lifecycle-livedata-core:2.2.0 (*)
|    |    |    +--- androidx.lifecycle:lifecycle-viewmodel:2.2.0 (*)
|    |    |    \--- androidx.lifecycle:lifecycle-viewmodel-savedstate:1.0.0 (*)
|    |    \--- com.squareup.okhttp3:okhttp:3.12.3 -> 3.12.10 (*)
|    +--- com.google.android.material:material:1.1.0
|    |    +--- androidx.annotation:annotation:1.0.1 -> 1.1.0
|    |    +--- androidx.appcompat:appcompat:1.1.0
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    |    +--- androidx.cursoradapter:cursoradapter:1.0.0
|    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    +--- androidx.fragment:fragment:1.1.0 -> 1.2.0 (*)
|    |    |    +--- androidx.appcompat:appcompat-resources:1.1.0
|    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    +--- androidx.core:core:1.0.1 -> 1.2.0 (*)
|    |    |    |    +--- androidx.vectordrawable:vectordrawable:1.1.0
|    |    |    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    |    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    |    |    |    \--- androidx.collection:collection:1.1.0 (*)
|    |    |    |    +--- androidx.vectordrawable:vectordrawable-animated:1.1.0
|    |    |    |    |    +--- androidx.vectordrawable:vectordrawable:1.1.0 (*)
|    |    |    |    |    +--- androidx.interpolator:interpolator:1.0.0
|    |    |    |    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |    |    \--- androidx.collection:collection:1.1.0 (*)
|    |    |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    |    +--- androidx.drawerlayout:drawerlayout:1.0.0
|    |    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    |    |    +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    |    \--- androidx.customview:customview:1.0.0 (*)
|    |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.cardview:cardview:1.0.0
|    |    |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    +--- androidx.coordinatorlayout:coordinatorlayout:1.1.0
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    |    +--- androidx.customview:customview:1.0.0 (*)
|    |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    +--- androidx.fragment:fragment:1.0.0 -> 1.2.0 (*)
|    |    +--- androidx.lifecycle:lifecycle-runtime:2.0.0 -> 2.2.0 (*)
|    |    +--- androidx.recyclerview:recyclerview:1.0.0 -> 1.1.0
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |    |    +--- androidx.customview:customview:1.0.0 (*)
|    |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.transition:transition:1.2.0
|    |    |    +--- androidx.annotation:annotation:1.1.0
|    |    |    +--- androidx.core:core:1.0.1 -> 1.2.0 (*)
|    |    |    \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.vectordrawable:vectordrawable:1.1.0 (*)
|    |    \--- androidx.viewpager2:viewpager2:1.0.0
|    |         +--- androidx.annotation:annotation:1.1.0
|    |         +--- androidx.fragment:fragment:1.1.0 -> 1.2.0 (*)
|    |         +--- androidx.recyclerview:recyclerview:1.1.0 (*)
|    |         +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|    |         \--- androidx.collection:collection:1.1.0 (*)
|    +--- androidx.constraintlayout:constraintlayout:1.1.3
|    |    \--- androidx.constraintlayout:constraintlayout-solver:1.1.3
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.0 -> 1.4.10 (*)
|    +--- org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.0 (*)
|    +--- project :libnavigation-util (*)
|    +--- com.mapbox.mapboxsdk:mapbox-sdk-services:5.8.0-beta.2 (*)
|    +--- com.mapbox.mapboxsdk:mapbox-sdk-turf:5.8.0-beta.2 (*)
|    +--- com.mapbox.mapboxsdk:mapbox-android-plugin-annotation-v9:0.8.0
|    |    +--- androidx.appcompat:appcompat:1.0.0 -> 1.1.0 (*)
|    |    \--- com.mapbox.mapboxsdk:mapbox-android-sdk:9.0.0 -> 9.6.0-alpha.1 (*)
|    +--- androidx.lifecycle:lifecycle-extensions:2.2.0
|    |    +--- androidx.lifecycle:lifecycle-runtime:2.2.0 (*)
|    |    +--- androidx.arch.core:core-common:2.1.0 (*)
|    |    +--- androidx.arch.core:core-runtime:2.1.0 (*)
|    |    +--- androidx.fragment:fragment:1.2.0 (*)
|    |    +--- androidx.lifecycle:lifecycle-common:2.2.0 (*)
|    |    +--- androidx.lifecycle:lifecycle-livedata:2.2.0 (*)
|    |    +--- androidx.lifecycle:lifecycle-process:2.2.0
|    |    |    \--- androidx.lifecycle:lifecycle-runtime:2.2.0 (*)
|    |    +--- androidx.lifecycle:lifecycle-service:2.2.0
|    |    |    \--- androidx.lifecycle:lifecycle-runtime:2.2.0 (*)
|    |    \--- androidx.lifecycle:lifecycle-viewmodel:2.2.0 (*)
|    +--- com.squareup.picasso:picasso:2.71828
|    |    +--- com.squareup.okhttp3:okhttp:3.10.0 -> 3.12.10 (*)
|    |    +--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    |    \--- androidx.exifinterface:exifinterface:1.0.0
|    |         \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|    +--- com.jakewharton.timber:timber:4.7.1
|    |    \--- org.jetbrains:annotations:16.0.1
|    +--- com.squareup.okhttp3:okhttp:3.12.10 (*)
|    +--- com.squareup.okhttp3:logging-interceptor:3.12.10 (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9 (*)
+--- com.mapbox.base:common:0.4.0 (*)
+--- com.mapbox.common:logger:0.4.0 (*)
+--- com.mapbox.mapboxsdk:mapbox-android-plugin-annotation-v9:0.8.0 (*)
+--- com.mapbox.mapboxsdk:mapbox-sdk-turf:5.8.0-beta.2 (*)
+--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.0 -> 1.4.10 (*)
+--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9 (*)
+--- androidx.core:core:1.2.0 (*)
+--- com.google.android.material:material:1.1.0 (*)
+--- androidx.appcompat:appcompat:1.1.0 (*)
+--- androidx.constraintlayout:constraintlayout:1.1.3 (*)
+--- androidx.preference:preference-ktx:1.1.0
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.40 -> 1.4.10 (*)
|    \--- androidx.preference:preference:1.1.0
|         +--- androidx.appcompat:appcompat:1.1.0 (*)
|         +--- androidx.core:core:1.1.0 -> 1.2.0 (*)
|         +--- androidx.fragment:fragment:1.1.0 -> 1.2.0 (*)
|         +--- androidx.recyclerview:recyclerview:1.0.0 -> 1.1.0 (*)
|         +--- androidx.annotation:annotation:1.1.0
|         \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
+--- com.google.android.gms:play-services-location:17.0.0
|    +--- com.google.android.gms:play-services-base:17.0.0
|    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    +--- androidx.fragment:fragment:1.0.0 -> 1.2.0 (*)
|    |    +--- com.google.android.gms:play-services-basement:17.0.0
|    |    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    |    +--- androidx.core:core:1.0.0 -> 1.2.0 (*)
|    |    |    \--- androidx.fragment:fragment:1.0.0 -> 1.2.0 (*)
|    |    \--- com.google.android.gms:play-services-tasks:17.0.0
|    |         \--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    +--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    +--- com.google.android.gms:play-services-places-placereport:17.0.0
|    |    \--- com.google.android.gms:play-services-basement:17.0.0 (*)
|    \--- com.google.android.gms:play-services-tasks:17.0.0 (*)
+--- androidx.multidex:multidex:2.0.0
+--- com.jakewharton.timber:timber:4.7.1 (*)
+--- com.jakewharton:butterknife:10.1.0
|    \--- com.jakewharton:butterknife-runtime:10.1.0
|         +--- com.jakewharton:butterknife-annotations:10.1.0
|         |    \--- androidx.annotation:annotation:1.0.0 -> 1.1.0
|         \--- androidx.core:core:1.0.0 -> 1.2.0 (*)
+--- com.mapbox.mapboxsdk:mapbox-sdk-services:5.8.0-beta.2 (*)
\--- com.squareup.okhttp3:okhttp:3.12.10 (*)
```

30 matches here 😅 

In any case, if you have a strong opinion I can change it.

Reply:
oh there))) yeah. I would remove `Native` because it's not a native, right?

Reply:
Yeah, `mapbox-sdk-common` it's a native library 👍

Reply:
ok than, lets keep as it is